src/confluent_kafka/src/Consumer.c

```diff
@@ -1477,7 +1507,23 @@ static PyMethodDef Consumer_methods[] = {
           "send_offsets_to_transaction() API.\n"
           "\n"
         },
-
+        {
+            "io_event_enable",
+            (PyCFunction) Consumer_io_event_enable,
+            METH_VARARGS | METH_KEYWORDS,
+            ".. py:function:: io_event_enable(fd, payload)\n"
+            "\n"
+            "This method basically calls ``rd_kafka_queue_io_event_enable``\n"
```

Comment:
We try to abstract the underlying C library, so this doesn't tell the user much, so skip this line altogether.